apiserver/bundle/bundle_test.go

```diff
@@ -1,26 +1,47 @@
-// Copyright 2015 Canonical Ltd.
+// Copyright 2016 Canonical Ltd.
 // Licensed under the AGPLv3, see LICENCE file for details.
 
-package client_test
+package bundle_test
 
 import (
 	jc "github.com/juju/testing/checkers"
 	gc "gopkg.in/check.v1"
 
+	"github.com/juju/juju/apiserver/bundle"
+	"github.com/juju/juju/apiserver/common"
 	"github.com/juju/juju/apiserver/params"
+	apiservertesting "github.com/juju/juju/apiserver/testing"
+	jujutesting "github.com/juju/juju/juju/testing"
 )
 
-func (s *serverSuite) TestGetBundleChangesBundleContentError(c *gc.C) {
-	args := params.GetBundleChangesParams{
+type bundleSuite struct {
+	jujutesting.JujuConnSuite
```

Comment:
:x: This doesn't appear to actually need to be a jujuConnSuite... newFacade doesn't actually use s.State, for example.  We try to avoid making new suites that depend on JujuConnSuite because it's so slow.